wa_lap_identification.sql

```diff
@@ -0,0 +1,240 @@
+/***********************************************************************************************************************
+This query produces a roster of currently enrolled students at Atlas, Olympus, and Sierra and indicates whether they are LAP
+eligible. LAP eligibility is based on meeting the following criteria:
+
+-------
+ 
+LAP MATH:
+
+Bottom 20th percentile  (< 20)on MAP Math in most recent test administration for that student
+(considering current and previous school year; if student took test more than once in most recent administration, use highest score)
+
+AND
+
+<= C+ in Math in the previous year
+
+-------
+
+LAP ELA:
+
+Bottom 20th percentile (< 20) on MAP Reading in most recent test administration for that student
+(considering current and previous school year; if student took test more than once in most recent administration, use highest score)
+
+AND
+
+<= C+ in English in the previous year
+
+-------
+
+To update this query for a new school year, update the following:
+ - session ids in the student set table
+ - NWEA tables
+ - academic year in the grades tables
+ **********************************************************************************************************************/
+
+SELECT
+  student_set.site_name
+  , student_set.local_student_id
+  , last_name
+  , first_name
+  , student_set.grade_level
+  , map_math.map_percentile AS min_map_math_percentile
+  , map_math.map_term AS min_map_math_term
+  , math_grade
+  , CASE WHEN map_math.map_percentile < 20 THEN TRUE
+         WHEN math_grade IN ('C+', 'C', 'C-', 'D+', 'D', 'D-', 'F', 'INCOMPLETE') THEN TRUE
+         ELSE FALSE
+    END AS lap_math
+  , map_reading.map_percentile AS min_map_reading_percentile
+  , map_reading.map_term AS min_map_reading_term
+  , ela_grade
+  , CASE WHEN map_reading.map_percentile < 20 THEN TRUE
+         WHEN ela_grade IN ('C+', 'C', 'C-', 'D+', 'D', 'D-', 'F', 'INCOMPLETE') THEN TRUE
+         ELSE FALSE
+    END AS lap_ela   
+
+
+FROM
+  /* Student set: currently enrolled students at Atlas, Olympus, and Sierra */
```

Comment:
We actually want students enrolled not currently but rather enrolled in the fall.  I think we can choose a date in the fall like 10/4.

Reply:
Agreed on doing 10/1